src/css/profile/mobile/common/listview.less

```diff
@@ -140,25 +140,34 @@ tau-expandable {
 			}
 		}
 
-		&.ui-li-has-icon {
-			margin-left: 12 * @px_base;
-		}
-
 		// Text
 		.ui-li-text {
 			display: inline-flex;
 			flex-direction: column;
-			padding: 14 * @px_base 0;
+			padding: 15 * @px_base 0;
 			margin-right: 24 * @px_base;
 
 			.ui-li-text-title {
 				font-size: 18 * @sp_base;
 				color: var(--text-color);
+				line-height: 21 * @px_base;
+
+				&:only-child {
+					padding-bottom: @px_base;
```

Comment:
There is no value. Is it only intended to fill in the units?

Reply:
@px_base == 1px